### ISSUE TITLE
Problem of git clone this repository using lfs

### ISSUE THREAD

**Issue by hanlinniu** [NVIDIA/Dataset_Synthesizer#63]
Error downloading object: Documentation/NDDS.pdf (25eb4ca): Smudge error: Error downloading Documentation/NDDS.pdf (25eb4cab49762b72f6fb6eff70775c4baea7892fac6a76b237ce35bcb578e2dd): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

**Comment by thangt** [comment]
@hanlinniu: Sorry for the trouble! I just posted an update and attach the compressed code there: https://github.com/NVIDIA/Dataset_Synthesizer/releases/download/1.2.2/ndds_1.2.2.zip.
Please try it out and let me know if you have any other troubles with it!

**Comment by hanlinniu** [comment]
@thangt , thank you very much! Does the ndds 1.2.2 work with unrealengine 4.21 or I have to use unrealengine 4.22?

**Comment by thangt** [comment]
It should be UE 4.22. I don't think it will work with 4.21.

**Comment by hanlinniu** [comment]
@thangt , ok, get it, thanks, I just found in Documentation NDDS.pdf, it said it works with UE4.21. maybe this document is for ndds 1.21 instead of ndds 1.22

**Comment by thangt** [comment]
Ah, sorry. We forgot to update the documentation. The NDDS should still work the same, the main different between UE 4.21 and UE 4.22 that concern the NDDS is the underlying code, not the UI.

**Comment by ShrutheeshIR** [comment]
Hi @thangt 
The zip file that you have linked is only the linux source code?
Where would I find a similar windows code?
I do not have ubuntu 16 (I have only ubuntu 18). Thus, I am running UE4 on windows. Please let me know what I can do !

**Comment by avinashsen707** [comment]
Its better to use on ubuntu 16.04.
But yet you can use in windows which will show some faster experience.

If you want it in win, then you must follow the requirements in .pdf and the versions too

**Comment by ShrutheeshIR** [comment]
Yes, since the lfs is broken I was wondering if you could share the zip file for windows as well? 
If it is the same for windows, could you please give instructions on how to compile? Because in this zip file that you'd shared, only the .sh files are there, which correspond to linux file systems.

Thanks
